_layouts/bare.html

```diff
@@ -21,8 +21,25 @@
     {% include nav.html %}
   </div>
 
-  <div class="bare-content" role="main" itemscope itemprop="mainContentOfPage">
-    {{ content }}{% include edit_link.html %}
+  <div ng-app="hubSearch" ng-controller="SearchController" class="bare-content" role="main" itemscope itemprop="mainContentOfPage">
+    {% raw %}
+      <div>
+        <live-search id="search1" type="text"
```

Comment:
The use of an id here concerns me. Why is there a "1"? What if another search for has to be included on the page? Could this be made modular instead?